Directory.Build.props

```diff
@@ -4,16 +4,9 @@
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
-    <!--CS1570: XML comment has badly formed XML 'Expected an end tag for element [parameter] -->
-    <!--CS1571: XML comment on [construct] has a duplicate param tag for [parameter] -->
-    <!--CS1572: XML comment has a param tag for '[parameter]', but there is no parameter by that name -->
-    <!--CS1573: Parameter has no matching param tag in the XML comment -->
-    <!--CS1574: XML comment has cref attribute that could not be resolved-->
-    <!--CS1734: XML comment has a paramref tag, but there is no parameter by that name-->
-    <WarningsAsErrors>nullable,CS1570,CS1571,CS1572,CS1573,CS1574,CS1734</WarningsAsErrors>
+    <WarningsAsErrors>true</WarningsAsErrors>
```

Comment:
Open to discussion, but would be nice if we do not have warnings at all

Reply:
From a historic context, we didn't do that because we want this to be a friendly first-open-source project for community members and a project that you build without too many issues. In terms of code, this can be a little bit ugly, because when we have to use some obsolete API we need to add the #prgma excluding the warning/error

Reply:
ok, reverted these changes

Reply:
Thanks for reverting this, Vlad!

Microsoft requires us to have the XML documentation (long story short, it's used to auto-generate API docs), so I added these `WarningsAsErrors` to ensure that future PRs always contain proper XML documentation 💯

Reply:
actually, I set it as true for all warnings including XML summary, but as Pedro suggested, to have an easier entry point for new contributors, I reverted it to the current settings